client/src/app/(onboarding)/login/page.tsx

```diff
@@ -1,3 +1,195 @@
+          variant="h4"
+          sx={{
+            fontWeight: "bold",
+            fontSize: "1.25rem",
+            textAlign: "center",
+          }}
+        >
+          Or
+        </Typography>
+        <Divider
+          sx={{
+            width: "100px",
+            height: "1px",
+            backgroundColor: "#E0E4F2",
+            marginLeft: 2,
+          }}
+        />
+      </div>
+
+      <div className="login-box">
+        <div className="login-fields">
+          <Typography
+            variant="h5"
+            sx={{
+              textAlign: "start",
+              width: "500px",
+              marginBottom: "10px",
+            }}
+          >
+            Email
+          </Typography>
+          <TextField
+            variant="outlined"
+            required
+            error={error}
+            helperText={emailHelperText}
+            autoComplete="email"
+            name="email"
+            label="Enter your email"
+            type="email"
+            value={email}
+            onChange={(e) => setEmail(e.target.value)}
+            sx={{
+              width: "500px",
+              "& .MuiOutlinedInput-root": {
+                "& fieldset": {
+                  borderColor: "#769FCD",
+                },
+              },
+            }}
+          />
+          <Typography
+            variant="h5"
+            sx={{
+              marginTop: "10px",
+              textAlign: "start",
+              width: "500px",
+              marginBottom: "10px",
+            }}
+          >
+            Password
+          </Typography>
+          <TextField
+            variant="outlined"
+            required
+            error={error}
+            helperText={passwordHelperText}
+            autoComplete="current-password"
+            name="password"
+            label="Enter your password"
+            type="password"
+            value={password}
+            onChange={(e) => setPassword(e.target.value)}
+            sx={{
+              width: "500px",
+              marginBottom: "30px",
+              "& .MuiOutlinedInput-root": {
+                "& fieldset": {
+                  borderColor: "#769FCD",
+                },
+              },
+            }}
+          />
+        </div>
+
+        <div className="login-button-links">
+          <div className="spacer"></div>
+          <div className="login-button">
+            <Button
+              variant="contained"
+              onClick={validateInput}
+              sx={{
+                backgroundColor: "#496FFF",
+                borderRadius: "8px",
+                padding: "10px 60px",
+                fontSize: "18px",
+                "&:hover": {
+                  backgroundColor: "#3C435C",
+                },
+              }}
+            >
+              <Typography sx={{ color: "white" }}>Log In</Typography>
+            </Button>
+          </div>
+
+          <div className="links">
+            <Link href="/signup">
```

Comment:
The link styles are not consistent of the ones in the signup page
![Screenshot 2024-10-07 at 5 44 01 PM](https://github.com/user-attachments/assets/e79e8fd7-c9d1-47f4-86da-d67821b628fb)